robot/basestation/static/css/themes/lofi.css

```diff
@@ -0,0 +1,195 @@
+/*
+Bootstrap Buttons (Purple): #445B7B
+Regular Buttons (Green): #3D7F7F
+Table (Red): #C34641
+Table (Light Grey): #EFEFEF
+Navbar (Salmon): #A38282
+Text (Off-White): #E4E1D1
+Background (Grey): #2B343B
+Button Hover (Blueish Light Grey): #BCD4DD
+*/
+
+/* Background color */
+html, body {
+  height: 100%;
+  background-color: #2B343B;
+}
+
+/* Navigation bar color */
+.navbar {
+  background-color: #A38282;
+}
+
+/* Whole lotta bootstrap dropdown menu styling */
+.dropdown .nav-link {color:#445B7B; text-decoration: none;}
+.dropdown .btn {background: #445B7B; color:#fff;}
+.container .dropdown .dropdown-menu a:hover
+{
+  color: #fff;
+  background-color: #fff;
+  border-color: white;
+}
+.btn-primary.dropdown-toggle {
+    color: #fff;
+    border-color: #fff;
+}
+.bootstrap-select .dropdown-toggle:focus {
+  outline: none!important;
+  box-shadow:
+  0 0 0 0.2rem rgba(0,0,0,.5);
+}
+.btn-primary.focus, .btn-primary:focus {
+  box-shadow:
+  0 0 0 0.2rem rgba(255,255,255,0);
+}
+.btn-primary:not(:disabled):not(.disabled).active, .btn-primary:not(:disabled):not(.disabled):active, .show>.btn-primary.dropdown-toggle {
+  color: #fff;
+  background-color: #445b7b;
+  border-color: #445b7b;
+}
+.btn-primary:not(:disabled):not(.disabled).active:focus, .btn-primary:not(:disabled):not(.disabled):active:focus, .show>.btn-primary.dropdown-toggle:focus {
+  box-shadow: 0 0 0 0.2rem rgba(255,255,255,0);
+}
+.btn.focus, .btn:focus {
+  box shadow: white;
+}
+.btn-primary {
+  border-color: white;
+}
+.dropdown-item.active, .dropdown-item:active {
+    color: #fff;
+    text-decoration: none;
+    background-color: #BCD4DD;
+}
+
+/* Bootstrap buttons */
+.btn-primary {
+  color: #fff;
+  background-color: #445b7b;
+  border-color: #445b7b;
+}
+.btn-primary:hover {
+  color: black;
+  background-color: #BCD4DD;
+  border-color: #BCD4DD;
+}
+
+/* Bootstrap forms */
+.form-control:focus {
+  color: white;
+  background-color: #fff;
```

Comment:
I see that sometimes lower case and sometimes upper case are used for these hex values. I think it would be best to keep it consistent with one or the other. We haven't made that decision as a team yet but my opinion would be to keep it all lower case for the reason of less total key strokes.

Reply:
I think it might be best to do all uppercase since hex values copied from adobe kuler palettes are uppercase. It'd save time on having to re-write the hex values.

Reply:
Fair enough